solutions/usaco-787.mdx

```diff
@@ -87,4 +86,85 @@ int main() {
 ```
 
 </CPPSection>
+<JavaSection>
+```java
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.StringTokenizer;
+
+public class rental {
+    static class Shop {
```

Comment:
tabs instead of spaces

Reply:
Um that's a tab (before "static class Shop").  Are you referring to something else?

Reply:
no it isn't?

Reply:
Your entire code uses spaces instead of tabs, see: https://usaco.guide/general/adding-solution#general